pkg/pillar/evetpm/tpm.go

```diff
@@ -119,6 +119,109 @@ var (
 	// measurefsTpmEventLog is the file containing the event log from the measure-config.
 	// it is not a constant due to test usage.
 	measurefsTpmEventLog = types.PersistStatusDir + "/measurefs_tpm_event_log"
+
+	// PCRSelection is used as an entropy to generate keys and the selection
```

Comment:
Name in comment doesn't match name of variable.

Reply:
fixed.